### ISSUE TITLE
Lots of Error as soon as its build

### ISSUE THREAD

**Issue by Mr4x3** [jshiohaha/solana-cookie-jar#1]
after building got error, cant start demo
![Screenshot from 2022-02-01 20-05-06](https://user-images.githubusercontent.com/9283776/151988277-8cf5a284-cd0f-4aad-a493-3c1af25986e4.png)

**Comment by jshiohaha** [comment]
This error is due to the fact that the demo is referencing the local version of `solana-cookie-jar`, which can be found in the demo's [package.json](https://github.com/jshiohaha/solana-cookie-jar/blob/main/demo/package.json#L25). To fix the errors you see, you will need to follow the instructions in [this StackOverflow post](https://sm-stackoverflow.azurefd.net/questions/56021112/react-hooks-in-react-library-giving-invalid-hook-call-error/57422196) to link react and react-dom packages. Then, you can make changes to the local solana-cookie-jar project.

Alternatively, if you don't want to reference the local version of the project, remove that line, run `npm i solana-cookie-jar`, and then run `npm run build` from inside the demo project.

**Comment by Mr4x3** [comment]
tried that too, start to give dependency conflics with other packages.

**Comment by jshiohaha** [comment]
What dependency conflicts are you seeing? I just pulled the repo to a new computer, did the setup again, and everything works as expected.

Here are the commands I ran:

```
git clone https://github.com/jshiohaha/solana-cookie-jar.git
cd solana-cookie-jar
yarn
npm link react
npm link react-dom
cd demo
npm i && npm run build
npm link react
npm link react-dom
npm run start
```

Also, make sure you add a `.env` file or directly set a public key [here](https://github.com/jshiohaha/solana-cookie-jar/blob/cb4d1074ffcbe738c1d9e06b7bc04a6ecca8d1f5/demo/src/components/Wallet/index.tsx#L10).
